src/Symfony/Component/Serializer/Tests/Normalizer/DateTimeNormalizerTest.php

```diff
@@ -272,6 +272,7 @@ public function denormalizeUsingTimezonePassedInContextProvider()
 
     /**
      * @expectedException \Symfony\Component\Serializer\Exception\UnexpectedValueException
+     * @expectedExceptionMessage DateTimeImmutable::__construct(): Failed to parse time string (invalid date) at position 0 (i): The timezone could not be found in the database. The DateTime for class "DateTimeInterface" is wrong ("invalid date" given).
```

Comment:
Maybe put the original exception message last?

`The DateTime for class "DateTimeInterface" is wrong: DateTimeImmutable::__construct(): Failed to parse time string (invalid date) at position 0 (i): The timezone could not be found in the database.`

Actually, not sure this bring much value at all for this denormalizer.